changelogs/current.yaml

```diff
@@ -22,6 +22,11 @@ behavior_changes:
     Fixed tag extraction so that :ref:`stat_prefix <envoy_v3_api_msg_extensions.filters.udp.dns_filter.v3.DnsFilterConfig>`
     is properly extracted. This changes the Prometheus name from
     dns_filter_myprefix_local_a_record_answers{} to dns_filter_local_a_record_answers{envoy.dns_filter_prefix="myprefix"}.
+- area: stats connection_limit
+  change: |
+    Fixed tag extraction so that :ref:`stat_prefix <envoy_v3_api_msg_extensions.filters.network.connection_limit.v3.ConnectionLimit>`
+    is properly extracted. This changes the Prometheus name from envoy_connection_limit_myprefix_limited_connections{} to
+    envoy_connection_limit_limited_connections{envoy_connection_limit_prefix="myprefix"}.
```

Comment:
these should be marked as literals - ie

> ``envoy_connection_limit.myprefix_limited_connections{}`` to ``envoy_connection_limit.limited_connections{envoy_connection_limit_prefix="myprefix"}``.

Reply:
Thanks. I will add backticks for metric names.